app/views/support/locations/_form.html.erb

```diff
@@ -1,11 +1,10 @@
-<%= form_with model: [:support, site], local: true, url: form_url, class: "location-form" do |f| %>
-
+<%= form_with model: @location_form, local: true, url: form_url, class: "location-form" do |f| %>
   <%= f.govuk_error_summary %>
 
-  <%= f.govuk_text_field :location_name, label: { text: "Name", size: "m" } %>
+  <%= f.govuk_text_field :location_name, label: { text: "Name", size: "s" } %>
 
   <%= f.govuk_text_field(:urn,
-    label: { text: "Unique Reference Number (URN)", size: "m" },
+    label: { text: "Unique Reference Number (URN) (optional)", size: "s" },
```

Comment:
I think the URN should have a width of 10 too, similar to [this](https://github.com/DFE-Digital/publish-teacher-training/blob/f1aee75bd29f3c2710c104b91eb05ae705f7eb2f/app/views/support/providers/locations/new_multiple/show.html.erb#L30-L32) (although it doesn't currently in the proto).

<img width="631" alt="image" src="https://user-images.githubusercontent.com/50492247/226893330-dcf3ef2c-14f2-42a8-9c5c-1ea1abdea36c.png">

Reply:
Will go with the proto for now, see what Simon says?